src/config/default.yml

```diff
@@ -240,10 +129,3 @@ try:
       level: 'nss-try'
       scopes:
         - "assume:repo:hg.mozilla.org/projects/nss-try:*"
-
-    # This has a repository entry in the DB that was created manually. When this is removed,
-    # the database entry should be removed as well.
-    users/mozilla_hocat.ca/comm-taskcluster:
```

Comment:
(and I apologize -- I asked in irc when I started this bug and thought you said "no" but now that I double-check I was imagining that answer..)